our-blog.md

```diff
@@ -0,0 +1,50 @@
+# Our Blog
+
+We currently maintain a Blog on Medium called [AnitaB.org Open Source Medium Publication](https://medium.com/anitab-org-open-source).
```

Comment:
`a blog on Medium` or `a Medium blog`?

Reply:
or ```Publication on Medium``` / ```Medium Publication```

Reply:
I already had medium publication in the sentence, I decided to move it back. i'll commit shortly